packages/wheelhouse/src/developmentActions.js

```diff
@@ -47,3 +47,23 @@ export const developmentLog = (pkgName, text) => dispatch => {
 export const developmentBuild = () => async (dispatch, getState) => {
   await dispatch(configLoad());
 };
+
+export const developmentLint = () => async (dispatch, getState) => {
+  await dispatch(configLoad());
+
+  const { packages } = getState();
+  let errored = "";
+  for (let key in packages) {
+    if (packages[key].lintingError) {
+      errored += key;
+    }
+  }
+  /*eslint-disable no-console*/
+  if (errored.length > 0) {
+    throw new Error(
+      `package.json and Chart.yaml are out of sync in: ${errored}`
+    );
+  } else {
+    console.log("👍");
```

Comment:
you can use `developmentLog` here instead of `console.log` and avoid the linter problems